PowerQueryM/ObservedLoad.pq

```diff
@@ -0,0 +1,20 @@
+let
+    MCastDomain = "demo-gas.mea-analytics.tools",
+    MaxDate = DateTime.Date(DateTime.LocalNow()),
+    MinDate = Date.AddDays(MaxDate, -60),
+    ApiKey = "<your API Key here>"
```

Comment:
```suggestion
    ApiKey = "your API Key here"
```
I bet someone might keep the angle brackets in thinking that they were necessary.  I actually already got rid of the angle brackets in the forecasted load example, I just forgot to do so here too.